frontend/src/components/CriteriaBarChart.tsx

```diff
@@ -94,46 +95,48 @@ const CriteriaBarChart = ({ video }: Props) => {
     });
 
   return (
-    <BarChart width={500} height={500} layout="vertical" data={data}>
-      <XAxis
-        type="number"
-        domain={[BAR_CHART_CRITERIA_SCORE_MIN, BAR_CHART_CRITERIA_SCORE_MAX]}
-        hide={true}
-      />
+    <ResponsiveContainer width="100%" height={500}>
+      <BarChart layout="vertical" data={data}>
+        <XAxis
+          type="number"
+          domain={[BAR_CHART_CRITERIA_SCORE_MIN, BAR_CHART_CRITERIA_SCORE_MAX]}
+          hide={true}
+        />
 
-      <Bar dataKey="clipped_score" barSize={20}>
-        {data.map((entry) => (
-          <Cell
-            key={entry.criteria}
-            fill={criteriaColors[entry.criteria] || criteriaColors['default']}
-          />
-        ))}
-      </Bar>
-      <YAxis
-        type="category"
-        dataKey="criteria"
-        tick={renderCustomAxisTick}
-        interval={0}
-        axisLine={false}
-        tickLine={false}
-        width={6}
-      />
-      <Tooltip
-        cursor={{ stroke: 'black', strokeWidth: 2, fill: 'transparent' }}
-        content={(props: TooltipProps<number, string>) => {
-          const payload = props?.payload;
-          if (payload && payload[0]) {
-            const { criteria, score } = payload[0].payload;
-            return (
-              <pre>
-                {getCriteriaLabel(criteria)}: {displayScore(score)}
-              </pre>
-            );
-          }
-          return null;
-        }}
-      />
-    </BarChart>
+        <Bar dataKey="clipped_score" barSize={20}>
+          {data.map((entry) => (
+            <Cell
+              key={entry.criteria}
+              fill={criteriaColors[entry.criteria] || criteriaColors['default']}
+            />
+          ))}
+        </Bar>
+        <YAxis
+          type="category"
+          dataKey="criteria"
+          tick={renderCustomAxisTick}
```

Comment:
The method `renderCustomAxisTick` does not work well anymore, because it was using the hardcoded value `250` for the width of the graph. We should find a way to know the width of the compoennt and use this or maybe specify `left: 50%` somehow

![Screenshot 2022-04-22 at 16 39 35](https://user-images.githubusercontent.com/8818081/164737055-7bce0527-ff11-447e-b3fe-769a266bdc01.png)

Reply:
It should be fixed (0c4d5d4d2034326ce852d28d5e40a0e1e97afcc8).